src/main/java/com/actor/example/MultiplierActor.java

```diff
@@ -19,6 +19,12 @@ public int getMultiplier() {
         }
     }
 
+    private void TestMethod() {
+        List<Integer> list = new ArrayList<>();
+        list.add(1);
+        System.out.prinlln(list);
+        }
+
     @Override
     public Receive createReceive() {
```

Comment:
Typo in System.out.println method name. It should be "println" instead of "prinlln". This will cause a compilation error.
TestMethod() -> System.out.prinlln(list);